src/opentracing/tracer.js

```diff
@@ -65,6 +79,8 @@ class DatadogTracer extends Tracer {
   }
 
   _inject (spanContext, format, carrier) {
+    if (spanContext === this._noopSpan.context()) return this
```

Comment:
@palazzem Should we propagate a sampling priority of `USER_REJECT` when the trace is dropped by client sampling, or should we simply not propagate any tracing information which will result in any other services called to be new root traces?